lib/helpers.js

```diff
@@ -122,7 +120,7 @@ exports.generateRootCapability = async ({
   }
   // dynamically generate zcap for root capability
   return {
-    '@context': SECURITY_CONTEXT_V2_URL,
+    '@context': sec.constants.SECURITY_CONTEXT_V2_URL,
```

Comment:
I think it's safe to start using the zcap context here as well (instead of the security context). Same with pretty much all other instances of security -- we're completely replacing its use with zcap or webkms contexsts.

Reply:
This was implemented.